content/en/post/kangxingyuan_candar2023/index.md

```diff
@@ -0,0 +1,35 @@
+---
+# Documentation: https://wowchemy.com/docs/managing-content/
+
+title: "Miss Kang Xingyuan's research presentation at 2023 Eleventh International Symposium on Computing and Networking(CANDAR)"
+subtitle: ""
+summary: ""
+authors: ["xingyuan-kang"]
+tags: ["Distributed SDN","Controller placement problem (CPP)","Consensus Algorithm"]
```

Comment:
Add just "SDN" too so that it can be categorized with the other SDN-related articles.